charts/enterprise-kyverno-operator/README.md

```diff
@@ -0,0 +1,80 @@
+# Helm Chart for Enterprise Kyverno
+Enterprise Kyverno is a Kubernetes Operator to manage lifecycle of Kyverno, Adapters and Nirmata supported policies. 
+
+## Prerequisites
+### Get license key
+You need a license key to run Enterprise Kyverno. If you are using Nirmata Enterprise for Kyverno, it is available in the UI. Else contact support@nirmata.com.
+
+### Install cert-manager 
```

Comment:
Is this mandatory or optional step? I don't recollect I have ever run this. If it is optional, please mention that.

Reply:
Cert manager is new now, since we need it for webhooks. Previously those were not there. One webhook for recording user info, other(s) for enforcing some checks in CR (disallowing some changes).